pkg/kclient/binding.go

```diff
@@ -121,7 +122,45 @@ func NewServiceBindingObject(bindingName string, bindAsFiles bool, deploymentNam
 	}
 }
 
-func mappingContainsBKS(bindableObjects []*meta.RESTMapping, bks sboApi.BindableKindsStatus) bool {
+// GetBindingServiceBinding returns a ServiceBinding from group binding.operators.coreos.com/v1alpha1
+func (c Client) GetBindingServiceBinding(name string) (bindingApi.ServiceBinding, error) {
```

Comment:
Since it's returning a ServiceBinding from Binding, do you think it would be better to rename this to `GetServiceBindingFromBinding`? And then the `name` parameter is the name of the Binding using which the method tries to find the Binding.

Reply:
The prefix `Binding` here (as opposed to `Spec` in the following method), indicates the group of the ServiceBinding (binding.operators.coreos.com/v1alpha1 as opposed to servicebinding.io/v1alpha3 alias Spec)

I find this order more readable as the method
GetBindingServiceBinding
returns a struct
bindingApi.ServiceBinding

and
GetSpecServiceBinding
returns 
specApi.ServiceBinding